day_7/fizzbuzz.js

```diff
@@ -0,0 +1,24 @@
+/* make a loop for 1-100 where the reminder if 0 will log fizzbuz,
+buzz or fiz, */
+fizzBuzznumber(-100);
+var numberPlus = (var i = 0; i <= number; i++)
+var numberPlus = (var i = 0; i >= number; i--)
+
+function fizzBuzznumber(number){
+
+for (var i = 0; increment; incrementPlus) {
```

Comment:
@MichaelEWalker87 - For your `for` loop on line 9, where are you declaring the variables `increment` and `incrementPlus`?